tailwind.config.ts

```diff
@@ -11,6 +11,56 @@ const config: Config = {
       colors: {
         background: 'var(--background)',
         foreground: 'var(--foreground)',
+        white: 'var(--white)',
+        black: 'var(--black)',
+        /**
+         * Gray
+         */
+        gray: {
+          50: 'var(--gray-50)',
+          75: 'var(--gray-75)',
+          100: 'var(--gray-100)',
```

Comment:
[Q]

오 잘 몰라서 그러는데, 혹시 CSS 변수로 한 번 매핑한 이유가 무엇인가요?

```suggestion
        gray: {
          50: '#f9f9fb',
          75: '#f1f1f4',
          100: '#dddee7',
```

요렇게 사용하는 것과 어떤차이가 있나용? (ex: 브라우저 캐싱 등? 🤔)

Reply:
@semnil5202 

`CSS 변수(var(--gray-XX))`와 `직접적인 HEX` 값 사용에는 몇 가지 차이가 있습니다.

### CSS 변수 사용 (var(--gray-XX))

* 동적 테마 변경 가능 - CSS 변수는 `:root`  또는 특정 요소에서 정의된 값을 기반으로 하기 때문에, 테마 변경이 필요한 경우 유용합니다. 즉) 다크 모드 등의 테마 변경에 유용합니다.
>  현재 디자인 토큰이 다 나오지는 않았지만, 추후 다크 모드도 대비하기 위해 사용하였습니다.

* 브라우저 최적화 - 브라우저는 CSS 변수를 캐싱하여 스타일 변경을 더 빠르게 적용할 수 있습니다.

### 직접 HEX 값을 사용한 경우
* 초기 렌더링 속도가 빠름: 값이 정적으로 정해져 있어 브라우저가 바로 적용 가능.
* 테마 변경이 어려움: 다크 모드 등 동적으로 색상을 바꿔야하는 경우 모든 색상을 직접 변경해야합니다.

직접 HEX 값을 사용하는것과 CSS 변수를 통해 다시 한번 매핑하는것은 위와 같은 차이점이 있습니다.

 개인적으로 색상 팔레트는 `:root` 에서 모아두어, 어디서든 참조 가능하게 하면서, 유지보수성의 일환으로 추후 디자인 색상이 변경 되어있을때 `:root`에서 변수만 변경하게 되면 모든 UI에도 반영 할 수 있기 때문에 한번 더 매핑에서 사용하는 것을 선호합니다.
 
 부가적으로 `shadcn-ui`를 사용해서 초기 팔레트 구성을 사용하게 되면, 위와 같이 매핑을 해서 사용하기 때문에 추후 shadcn-ui 도입을 염두해두고 일관성을 유지하기 위해 매핑해서 작성한 점도 있습니다.

Reply:
오우 이 야무진 코멘트를 놓치고 갔었군요.

> CSS 변수 캐싱 및 shadcn-ui 초기 팔레트 구성 유리

상세한 설명 덕분에 잘 이해했습니다! 🙏🙏😘